### ISSUE TITLE
Implement efficient airdrops in `split`

### ISSUE THREAD

**Issue by raphjaph** [ordinals/ord#4081]
This can either be done implicitly by recognizing that the `split.yaml` is airdropping the same amount to multiple outputs or explictly by introduction a `mode` in the split file called `even` or `airdrop`.

For reference #3658

**Comment by bingryan** [comment]
#3886

**Comment by kelven7224** [comment]
> #3886
>

**Comment by kelven7224** [comment]
> #3886
>

**Comment by dcorral** [comment]
What is the limit for a split? 
How many outputs (aprox) can you fit in one split.yml?

**Comment by raphjaph** [comment]
Probably around 9000 P2TR outputs with 2 P2TR inputs and an 83 byte OP_RETURN

https://bitcoinops.org/en/tools/calc-size/

**Comment by blieb** [comment]
Was testing it on testnet, when I added more than 5 it did not fit in op_return anymore

**Comment by dcorral** [comment]
> when I did it with 100 outputs: `error: runestone size 3351 over maximum standard OP_RETURN size 83`

Exactly.

What I really meant is how many explicit edicts can you add to a split file? I guess it depends on the amount size of the edicts?

**Comment by raphjaph** [comment]
> Probably around 9000 P2TR outputs with 2 P2TR inputs and an 83 byte OP_RETURN
> 
> https://bitcoinops.org/en/tools/calc-size/

This is wrong, I was assuming a max standard size of 400000 Bytes but it's actually 100000 Bytes. So I think that would be around 2300 outputs.


> What I really meant is how many explicit edicts can you add to a split file? I guess it depends on the amount size of the edicts?

At the moment the split file does one edict per entry so you reach the OP_RETURN standardness limit of 83 bytes after around 10 edict. If you do an even split you can however do less edicts, so fit into the limit. See https://github.com/ordinals/ord/issues/3658
